src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateDeconstructMethodTests.cs

```diff
@@ -127,13 +127,11 @@ void Method()
         (var x, var y) = [|this|];
     }
 }",
-@"using System;
-
-class Class
+@"class Class
 {
     private void Deconstruct(out object x, out object y)
     {
-        throw new NotImplementedException();
+        throw new System.NotImplementedException();
```

Comment:
this is the *correct* behavior.  we made the explicit decision we woudl not add an import for `throw new System.NotImplementedException();`.  the code that i fixed up broke this and caused us to still add that usign and simplify this.